tests/functional/test_action_config.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import pytest
+import allure
+
+from coreapi.exceptions import ErrorMessage
+
+from adcm_client.objects import ADCMClient, Cluster
+from adcm_pytest_plugin import utils as plugin_utils
+from adcm_pytest_plugin.steps.actions import run_cluster_action_and_assert_result
+
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture()
+def cluster(sdk_client_fs: ADCMClient) -> Cluster:
+    """Get cluster"""
+    uploaded_bundle = sdk_client_fs.upload_from_fs(plugin_utils.get_data_dir(__file__, "cluster"))
+    return uploaded_bundle.cluster_create('test_cluster')
+
+
+def test_config_not_presented(cluster: Cluster):
+    """
+    Action's configuration not presented
+    Run action with config
+    """
+    expected_message = (
+        "The task includes an option with an undefined variable. "
+        "The error was: 'dict object' has no attribute 'config'"
+    )
+    with allure.step('Run action without config and expect it to fail with ansible error'):
+        task = run_cluster_action_and_assert_result(cluster, 'no_config', 'failed', config={'some_param': 1})
```

Comment:
it should be 400 error on validating config

Reply:
Changed and added cluster/provider related object's actions